doozer/doozerlib/cli/scan_osh.py

```diff
@@ -0,0 +1,179 @@
+import click
+import koji
+import asyncio
+from doozerlib.cli import cli, click_coroutine, pass_runtime
+from doozerlib.runtime import Runtime
+from doozerlib.exectools import fire_and_forget, cmd_gather_async, limit_concurrency
+from doozerlib.util import cprint
+from typing import Optional
+
+
+class ScanOshCli:
+    def __init__(self, runtime: Runtime, last_brew_event: int, dry_run: bool, nvrs: Optional[list],
+                 check_triggered: Optional[bool], all_builds: Optional[bool]):
+        self.runtime = runtime
+        self.last_brew_event = last_brew_event
+        self.dry_run = dry_run
+        self.brew_tags = []
+        self.specific_nvrs = nvrs
+        self.check_triggered = check_triggered
+        self.all_builds = all_builds
+        self.error_nvrs = []
+
+        # Initialize runtime and brewhub session
+        self.runtime.initialize(clone_distgits=False)
+        self.koji_session = koji.ClientSession(self.runtime.group_config.urls.brewhub)
+
+    @staticmethod
+    async def get_untriggered_nvrs(nvrs):
+        """
+        So we might have already triggered a scan for the same NVR. If this flag is enabled, we want to check
+        if it has and not re-trigger it again.
+        But please note that this will take a lot of time since it has to run for all 200+ packages.
+        """
+
+        @limit_concurrency(50)
```

Comment:
Let's change this to 16, I see in other places 16/32 were the numbers chosen for multiple http calls, so that should be a safe number

Reply:
Buildsync is using 500. So this should be safe to keep right